addons/barcodes_gs1_nomenclature/models/barcode_rule.py

```diff
@@ -22,8 +22,8 @@ def _default_encoding(self):
             ('package', 'Package'),
             ('use_date', 'Best before Date'),
             ('expiration_date', 'Expiration Date'),
-            ('package_type', 'Packaging Type'),
-            ('packaging_date', 'Packaging Date'),
```

Comment:
I would use Package since Pack could be both Package and Packaging

Reply:
This was discussed earlier :arrow_up:  :wink: "Pack Date" sounds better in English/matches the "Put in Pack" language we already have. It's also sometimes used on food containers in the US already. Plus I already named it "Pack Date" in the datamatrix PR.